lorry/component/dbmanager.go

```diff
@@ -310,7 +310,7 @@ func (*FakeManager) JoinCurrentMemberToCluster(context.Context, *dcs.Cluster) er
 }
 
 func (*FakeManager) LeaveMemberFromCluster(context.Context, *dcs.Cluster, string) error {
-	return fmt.Errorf("NotSuppported")
+	return nil
```

Comment:
why no error messages ?

Reply:
For databases like MongoDB and WeSQL, handling members joining and leaving the cluster is necessary, but other databases do not require these actions. To support both cases, join and leave member actions are set to no-ops by default.